src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UpdateOtherPartyDataIt.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sscs.callback;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static uk.gov.hmcts.reform.sscs.helper.IntegrationTestHelper.assertHttpStatus;
+import static uk.gov.hmcts.reform.sscs.helper.IntegrationTestHelper.getRequestWithAuthHeader;
+
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletResponse;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class UpdateOtherPartyDataIt extends AbstractEventIt {
+
+    @Before
+    public void setup() throws IOException {
+        setup("callback/updateOtherPartyCallback.json");
```

Comment:
😭 Not everyone likes JSON. It's not _that_ many lines of code to create the appeal with other party data.
Ignore this comment, if you love this JSON file.

Reply:
I'll keep this in mind for the next time